typescript/examples/web-app/src/frontend/3dUtils/RealityDataCrs.ts

```diff
@@ -115,11 +115,11 @@ function realityModelFromJson(json: any, realityDataName: string, realityDataId?
     const format = realityDataType === "OPC" ? RealityDataFormat.OPC : RealityDataFormat.ThreeDTile;
     const prefix = process.env.IMJS_URL_PREFIX ?? "";
     const rdSourceKey: RealityDataSourceKey = realityDataId ? { provider, format, id: realityDataId } : RealityDataSource.createKeyFromUrl(
-        "https://" + prefix + "api.bentley.com/realitydata/" + realityDataId + "?projectId=" + process.env.IMJS_PROJECT_ID);
+        "https://" + prefix + "api.bentley.com/reality-management/" + realityDataId + "?projectId=" + process.env.IMJS_PROJECT_ID);
 
     const realityModel: ContextRealityModelProps = { 
         rdSourceKey, 
-        tilesetUrl: "https://" + prefix + "api.bentley.com/realitydata/" + realityDataId + "?projectId=" + process.env.IMJS_PROJECT_ID, 
+        tilesetUrl: "https://" + prefix + "api.bentley.com/reality-management/" + realityDataId + "?projectId=" + process.env.IMJS_PROJECT_ID, 
```

Comment:
This is wrong. Please look at https://developer.bentley.com/apis/reality-management/operations/get-all-reality-data/ projectId is not use with reality-management.